wdl/ReshardVcf.wdl

```diff
@@ -0,0 +1,87 @@
+version 1.0
+
+import "Structs.wdl"
+import "TasksMakeCohortVcf.wdl" as MiniTasks
+
+# Consumes a contig-sharded vcf where some shards may contain a small proportion of records from other
+#  contigs and moves them to their correct shards.
+
+workflow ReshardVcf {
+  input {
+    Array[File] vcfs  # Must be sorted and indexed
+    File contig_list
+    String prefix
+    Boolean? use_ssd
+    String sv_base_mini_docker
+    RuntimeAttr? runtime_override_reshard
+  }
+
+  Array[String] contigs = read_lines(contig_list)
+
+  scatter (i in range(length(vcfs))) {
+    File vcf_indexes = vcfs[i] + ".tbi"
+  }
```

Comment:
Would it be useful to add an optional input `Array[File]? vcf_indexe` and fall back to this approach if the optional input was not provided? I think that can be useful to hint to a user if indexes are needed, and also provide them with an option of providing the indexes explicitly if they are on a different path.

Reply:
As a convention we always require input VCFs to have indexes for top-level workflows. These are already generated by the pipeline so shouldn't cause any problems as long as the VCFs were not manually moved without their indexes.